{
    "command.error.internal": "Failed to process this command. {error}"
}